aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -2001,6 +2001,7 @@ TORCH_IMPL_FUNC(_linalg_solve_ex_out)(const Tensor& A,
   // Possible optimization: Compute the LU factorization of A^T if A is contiguous
   // Then we solve A^T X = B with adjoint=True
   // This saves a copy as A doesn't need to be copied into an F-contig matrix in lu_factor
+  // This optimization makes functorch's batching rule difficult, see comment in solve_ex_batch_rule
```

Comment:
nit: usually we mark notes as `NOTE [something]` and then refer to them as "see NOTE [something]"